src/infrastructure/persistence/PsqlStatisticRepository.ts

```diff
@@ -0,0 +1,26 @@
+import knex from 'knex';
+import { StatisticRepostory } from '../../domain/model/statistics/StatisticRepository';
+
+export class PsqlStatisticRepository implements StatisticRepostory {
+  constructor(private db: knex) {}
+
+  getTotalAmountOfUsers(): Promise<number> {
+    return this.count('user');
+  }
+
+  getTotalAmountOfTests(): Promise<number> {
+    return this.count('test');
+  }
+
+  getTotalAmountOfAccessPasses(): Promise<number> {
+    return this.count('access_pass');
+  }
+
+  private async count(table: string): Promise<number> {
+    const results: { count: string }[] = await this.db(table).count();
```

Comment:
Ooh, I see. If we're not persisting the statistics, why did you prefer a separate repository for it rather than adding a count method to each?

Reply:
I was basically copying the way reports were implemented, as I felt the domain is similar.